src/com/cft/contactmerge/tests/CompareContactPartsTest.java

```diff
@@ -252,6 +260,9 @@ void doPhoneNumbersMatch_No() {
         assertEquals(AnswerType.no, CompareContactParts.doPhoneNumbersMatch("(520) 123-4567", "(520) 123-4667"));
     }
 
+    /*******************************************************************************************************************
+     *********************************************** Compare Email Tests ***********************************************
+     *******************************************************************************************************************/
     @Test
     void doEmailsMatch_Yes_IgnoreSpaces() {
         assertEquals(AnswerType.yes, CompareContactParts.doEmailsMatch("jdoe@yahoo.com", " jdoe@yahoo.com "));
```

Comment:
Information coming in from the function "SplitStringUtilities" wouldn't have any white space in the front or after a record, and the "SplitStringUtilitiesTest" already tests this functionality.